doc/modules/feature_extraction.rst

```diff
@@ -400,7 +400,7 @@ Using stop words
 
 Stop words are words like "and", "the", "him", which are presumed to be
 uninformative in representing the content of a text, and which may be
-removed to avoid them being construed as signal for prediction.  Sometimes,
+removed to avoid them being construed as signals for prediction. Sometimes,
```

Comment:
From a purely grammatical point of view I think I agree with this change. But I wonder if in the context of machine-learning "signal" is the right thing here?